libflux/src/core/semantic/builtins.rs

```diff
@@ -628,11 +628,8 @@ pub fn builtins() -> Builtins<'static> {
                 "range" => r#"(
                         <-tables: [A],
                         start: B,
-                        ?stop: C,
-                        ?timeColumn: string,
-                        ?startColumn: string,
-                        ?stopColumn: string
-                    ) => [D] where A: Record, D: Record "#,
+                        ?stop: C
+                    ) => [A] where A: Record"#,
```

Comment:
I think `_start`, `_stop`, and `_time` need to be encoded directly within the type signature

    (<-tables: [{ T with _time: time }], start: A, ?stop: B) => [{ T with _time: time, _start: time, _stop: time }]

Reply:
Ah, that's much tighter. I'll remake the patch tomorrow using the `with _time...` components.